### PR TITLE
Update customize-templates.md

### PR DESCRIPTION
**Pull request by gregor-lohaus**
If another developer wouldnt have given me the information that i had to assign the theme to the sales channel i would have been completely lost

### CODE REVIEW

**Review by PaddyS — CHANGES_REQUESTED** [review]
Hey @gregor-lohaus,

thank you very much for your pull request! 🎉 

However I am not too sure about the changes. Not having assigned a theme to a sales channel will result in **no template** at all being shown, regardless of this guide.

While I do understand the concern, it doesn't seem to fit to this guide, since it's a problem anyway and not really bound to this guide, you know?

Anyway, I suggested a minor change, would be great if you could apply those suggested changes first. :-)
I will then have a little chat with a colleague in order to figure out if this is the proper place for such an information text.

**Comment by gregor-lohaus** [comment]
@PaddyS thanks for the review i updated the pull request with the requested changes

File: guides/plugins/plugins/storefront/customize-templates.md [hunk 1]
```diff
@@ -69,6 +69,12 @@ And that's it already, you're done. You might have to clear the cache and refres
 ```
 {% endtab %}
 
+{% hint style="info" %}
+Also remember to not only activate your plugin but also to assign your theme to the correct sales channel by clicking on it in the sidebar, going to the tab Theme and selecting your theme.
+{% endhint %}
+
+
+
```
Comment (PaddyS) [inline guides/plugins/plugins/storefront/customize-templates.md:77]:
```suggestion
```

**Comment by PaddyS** [comment]
Thank you @gregor-lohaus 🎉

**Pull request merged**